### PR TITLE
updating Mailjet code samples for Send API v3.1

### PR DESCRIPTION
**Pull request by adamyanliev**
Mailjet Send API v3.1 is being released, so updating the code samples to match the new Send API v3.1 configuration. Reference can be found [here](https://dev.mailjet.com/beta/).

### CODE REVIEW

File: appengine_flexible/mailjet/mailjet.go [hunk 1]
```diff
@@ -49,20 +49,26 @@ func sendEmail(w http.ResponseWriter, r *http.Request) {
...
+          },
+        },
+        Subject: "Your email flight plan!",
+        TextPart: "Dear passenger, welcome to Mailjet! May the delivery force be with you!",
+        HTMLPart: "<h3>Dear passenger, welcome to Mailjet!</h3><br />May the delivery force be with you!",
+      },
+    }
 
-	resp, err := mailjetClient.SendMail(m)
+	messages := mailjet.MessagesV31{Info: messagesInfo }
```
Comment (broady) [inline appengine_flexible/mailjet/mailjet.go]: Please run gofmt. The space before the } shouldn't be there, and there are spaces used for indentation above.

**Comment by broady** [comment]
```
appengine_flexible/mailjet/mailjet.go:78: resp.Sent undefined (type *mailjet.ResultsV31 has no field or method Sent)
```

(also, make sure to re-base/merge in master)

**Comment by adamyanliev** [comment]
@broady thanks for the help, updated now.

**Comment by tbpg** [comment]
Hey @adamyanliev, looks like this fell off the radar. Does this still make sense to merge?

**Comment by adamyanliev** [comment]
Hey @tbpg yes, seems that it was forgotten along the line. please merge, it is still relevant.

**Comment by tbpg** [comment]
Looks like there are some failures in Go 1.6 and 1.7:
> github.com/lib/pq
> ../../lib/pq/notify.go:787: undefined: time.Until

**Comment by adamyanliev** [comment]
This looks like an environment variable is missing, so the test can't run on 1.6 and 1.7. 

Looking at them, this is missing from the failing tests:

```
Setting environment variables from .travis.yml
$ export ALLOW_E2E=true
```

shouldn't be releated to the changes in the code samples

**Comment by tbpg** [comment]
Sorry - it was an unrelated dependency change (dropped support for Go < 1.8). I squashed all of the commits into one, updated the commit message, and tests are passing. Thank you!

**Comment by adamyanliev** [comment]
awesome, thanks for the help! :)

**Pull request merged**